src/main/java/org/reactome/release/qa/check/ChimericInstancesCheck.java

```diff
@@ -69,6 +74,9 @@ protected void checkChimericInstances(QAReport report, String clsName) throws Ex
             if (isEscaped(rle)) {
                 continue;
             }
+            if (skipList.inSkipList(rle)) {
+                continue;
```

Comment:
Is it possible to write this code without using `continue`?

Reply:
Done - the use of `continue` for the skiplist checks has been removed